base/reduce.jl

```diff
@@ -380,6 +380,10 @@ const ∈ = in
 ∋(itr, x)= ∈(x, itr)
 ∌(itr, x)=!∋(itr, x)
 
+# vectorized ∈ and ∉
+(.∈){T}(x::AbstractArray{T}, set) = [xx ∈ set for xx in x]
+(.∉){T}(x::AbstractArray{T}, set) = [xx ∉ set for xx in x]
```

Comment:
These should produce `BitArrays`; this should also do broadcasting the way other vectorized ops do.